app/workers/facilities/mental_health_reload_job.rb

```diff
@@ -35,26 +31,24 @@ def update_cache(model, records)
           local_updated: Time.now.utc.iso8601
         }
 
-        obj = model.find(r['StationNumber'])
-
-        if obj
-          obj.update(attrs)
-        else
-          model.create(attrs)
-        end
+        obj = FacilityMentalHealth.find_or_build(r['StationNumber'])
```

Comment:
Do we know anything more specific about the kinds of `obj` to give it a more meaningful name? E.g., just guessing, `station`.